plugin/include/cookie.h

```diff
@@ -11,26 +11,43 @@
 #include <string>
 #include <variant>
 
+#include <hilti/rt/fmt.h>
+
 #include <zeek-spicy/zeek-compat.h>
 
 namespace spicy::zeek::rt {
 
 namespace cookie {
 
+/** State stored inside protocol/file analyzer cookies to retain file analysis state. */
+struct FileState {
+    FileState(std::string analyzer_id) : analyzer_id(analyzer_id) {}
```

Comment:
```suggestion
    FileState(std::string analyzer_id) : analyzer_id(std::move(analyzer_id)) {}
```

(Requires `#include <utility>`).